client/src/views/pages/you-are-banned-what-now.js

```diff
@@ -0,0 +1,486 @@
+
+    return (
+        <div>
+            <div className={'text-center'}>
+                <h3>You are done!</h3>
+                <hr />
+                <p>We hope the previous shared information helps you resolve your issues.</p>
+                <p>If this is not the case please do contact us from our Discord server.</p>
+            </div>
+            <Stats step={6} {...props} />
+        </div>
+    );
+};
+
+
+export default function (props) {
+    const search = props.match.params.uid;
+    let isRedirected = false;
+    if(search === '11111111111111111') {
+        isRedirected = true;
+    }
+    const isValidSteam64ID = search && search.match(/^[0-9]{17}$/);
+
+    const {data} = isValidSteam64ID
+    ? useQuery(GET_PLAYER, { variables: { id: search } })
+    : { data: null };
+
+  return (
+    <Layout>
+      <section className="section section-lg pt-lg-0 mt--200">
+        <Container>
+          <Card className="shadow border-0">
+            <CardBody className="pt-5 pb-2 border-bottom">
+              <div className="icon icon-shape bg-gradient-warning rounded-circle text-white mb-4">
+                <i className="fa fa-life-ring" />
+              </div>
+              <h6 className="text-warning text-uppercase">SCBL For Dummies</h6>
+              <p className="description mt-2">
+                If you can't find what you search, please join our{' '}
+                <a href={DISCORD_INVITE} target="_blank" rel="noopener noreferrer">
+                  Discord
+                </a>{' '}
+                server.
+              </p>
+            </CardBody>
+            <div style={{ display: 'none' }}>
+                <AdvancedModal isOpen={isRedirected}>
+                   {(modal) => (
+                     <>
+                       <a
+                         href="/test"
+                         onClick={(e) => {
+                           e.preventDefault();
+                           modal.open();
+                         }}
+                       >
+                       <button className='mt-2 mb-2 btn btn-info btn-block'>How does threshold work?</button>
+                       </a>
+                       <Modal
+                         className="modal-dialog-centered"
+                         isOpen={modal.isOpen}
+                         toggle={modal.close}
+                       >
+                         <ModalHeader>Fill in your Steam64ID!</ModalHeader>
+                         <ModalBody className="text-center">
+                           <h3 className='text-primary'>Your Steam64ID:</h3>
+                           <SteamUserSearchBox uid={props.match.params.uid} />
+                           <p><small>You can find yours <a href='https://steamidfinder.com/lookup/' target='_blank' rel='noreferrer'>here</a>.</small></p>
+                         </ModalBody>
+                       </Modal>
+                     </>
+                   )}
+                 </AdvancedModal>
```

Comment:
Not sure a modal is best for this as it can easily be clicked out of by mistake.

I think a slide that conditionally shows based on the presence of a Steam user ID would bee better.